bin/test_travis.sh

```diff
@@ -98,8 +101,8 @@ else
     if [[ "${WITH_ARB}" != "" ]]; then
         extra_libs="$extra_libs -larb -lflint"
     fi
-    ${CXX} -std=c++0x -I$our_install_dir/include/ -L$our_install_dir/lib test_basic.cpp -lcsympy -lgmpxx -lgmp -lteuchos $extra_libs
-    export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+    ${CXX} -std=c++0x -I$our_install_dir/include/ -L$our_install_dir/lib test_basic.cpp -lcsympy -lteuchos -lgmpxx -lgmp $extra_libs
```

Comment:
It's not a big deal, but why don't you move the `-lteuchos` to where it was (behind `-lgmp`), to reduce clutter in the diff?